qiskit/circuit/controlflow/builder.py

```diff
@@ -205,10 +205,12 @@ class InstructionPlaceholder(Instruction, abc.ABC):
         with qc.for_loop(range(5)):
             qc.h(0)
             qc.measure(0, 0)
-            qc.break_loop().c_if(0, 0)
+            with qc.if_test((0, 0)):
+                qc.break_loop()
 
-    since ``qc.break_loop()`` needs to return a (mostly) functional
-    :obj:`~qiskit.circuit.Instruction` in order for :meth:`.InstructionSet.c_if` to work correctly.
+    ``qc.break_loop()`` needs to return a (mostly) functional
+    :obj:`~qiskit.circuit.Instruction` in order for the historical ``.InstructionSet.c_if``
+    to work correctly.
```

Comment:
was this only necessary for c_if? or is it also important for if_test? If it was only for c_if I would at least speak about it in past tense, given that it's dead:

```suggestion
    ``qc.break_loop()`` needed to return a (mostly) functional
    :obj:`~qiskit.circuit.Instruction` in order for the historical ``.InstructionSet.c_if``
    to work correctly.
```